modules/unauthenticated-users-cluster-role-binding-con.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="unauthenticated-users-cluster-role-bindings-concept_{context}"]
+= Cluster role bindings for unauthenticated groups
+
+There are use cases where it may be necessary to add `system:unauthenticated` to a cluster role.
```

Comment:
I am a may/might absolutist. ;-)
```suggestion
There are use cases where it might be necessary to add `system:unauthenticated` to a cluster role.
```